subsys/bluetooth/controller/ll_sw/ull_sync_types.h

```diff
@@ -57,3 +54,34 @@ struct node_rx_sync {
 	uint16_t interval;
 	uint8_t  sca;
 };
+
+#if defined(CONFIG_BT_CTLR_SYNC_ISO)
```

Comment:
I thought we stopped guarding struct declarations, based on the issues we've seen recently?